docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -268,10 +268,10 @@ GET /_data_stream/my-data-stream
 
 To delete a data stream and its backing indices, open the {kib} menu and go to
 *Stack Management > Index Management*. In the *Data Streams* tab, click the
-trash can icon.
+trash can icon. The availability of the trash icon corresponds to <<security-privileges, security privileges>> of your user role.
```

Comment:
Few suggestions:

- Change the original "trash can icon" to "trash icon" for consistency. This will also better align with the Kibana docs.
- I'd explicitly list the privilege needed for the icon to display. It looks like that's just `delete_index`.


```suggestion
trash icon. The trash icon only displays if you have the `delete_index`
<<security-privileges, security privilege>> for the stream.
```